app/test_split_integer.py

```diff
@@ -2,20 +2,38 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        sum(split_integer(10, 3)) == 10
+    ), "Sum of the parts should be equal to value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    parts = split_integer(6, 3)
```

Comment:
Don't create unnecessary variables for simple tests. Read checklist before pushing your solution